CHANGELOG.md

```diff
@@ -47,6 +47,7 @@
 - **General:** Improve e2e tests reliability ([#2580](https://github.com/kedacore/keda/issues/2580))
 - **General:** Syncronize HPA annotations from ScaledObject ([#2659](https://github.com/kedacore/keda/pull/2659))
 - **Memory Scaler** Adding e2e test for the memory scaler ([#2220](https://github.com/kedacore/keda/issues/2220))
+- **General:** Fix mismatched errors for updating hpa ([#2719](https://github.com/kedacore/keda/issues/2719))
```

Comment:
Just a minor nit, could you please move this item above `**Memory Scaler** ` so the list is sorted alphabetically.